.Lib9c.Tools/Utils.cs

```diff
@@ -7,18 +7,150 @@
 using System.Text.Json;
 using System.Text.Json.Serialization;
 using System.Threading.Tasks;
+using Cocona;
 using Libplanet;
 using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Blockchain.Policies;
 using Libplanet.Blocks;
 using Libplanet.Crypto;
+using Libplanet.RocksDBStore;
+using Libplanet.Store;
+using Libplanet.Store.Trie;
 using Nekoyume.Action;
+using Nekoyume.BlockChain;
 using Nekoyume.Model;
 using Nekoyume.Model.State;
+using Serilog;
+using Serilog.Core;
+using NCAction = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
 
 namespace Lib9c.Tools
 {
     public static class Utils
     {
+        public static Logger ConfigureLogger(bool verbose)
+        {
+            var logConfig = new LoggerConfiguration();
+            if (verbose)
+            {
+                logConfig = logConfig.WriteTo.Console();
+            }
+            return logConfig.CreateLogger();
+        }
+
+        public static (BlockChain<NCAction> Chain, IStore Store) GetBlockChain(
+            ILogger logger,
+            string storePath,
+            bool monorocksdb = false,
+            Guid? chainId = null
+        )
+        {
+            var policySource = new BlockPolicySource(logger);
+            IBlockPolicy<NCAction> policy = policySource.GetPolicy(
+                BlockPolicySource.DifficultyBoundDivisor + 1,
+                int.MaxValue
+            );
+            IStagePolicy<NCAction> stagePolicy = new VolatileStagePolicy<NCAction>();
+            IStore store
+                = monorocksdb
+                ? (IStore)new MonoRocksDBStore(storePath)
+                : new RocksDBStore(storePath);
+            IKeyValueStore stateKeyValueStore =
+                new RocksDBKeyValueStore(Path.Combine(storePath, "states"));
+            IKeyValueStore stateHashKeyValueStore =
+                new RocksDBKeyValueStore(Path.Combine(storePath, "state_hashes"));
+            IStateStore stateStore = new TrieStateStore(stateKeyValueStore, stateHashKeyValueStore);
+            Guid chainIdValue
+                = chainId is {} i
+                ? i
+                : store.GetCanonicalChainId() is {} cc
+                ? cc
+                : throw new CommandExitedException(
+                    "No canonical chain ID.  Available chain IDs:\n    " +
+                        string.Join("\n    ", store.ListChainIds()),
+                    1);
```

Comment:
How about using `??` keyword? Like:

```suggestion
            Guid chainIdValue
                = chainId ??
                  store.GetCanonicalChainId() ??
                  throw new CommandExitedException(
                      "No canonical chain ID.  Available chain IDs:\n    " +
                      string.Join("\n    ", store.ListChainIds()),
                      1);
```